spec/features/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  describe 'show page' do
+    before :each do
+      # @user = User.create(name: 'Tom', bio: 'Teacher from Mexico.', photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+      # @id = @user.id
+      # Post.create(user_id: @user, title: 'Hello', text: 'Make a new comment on this post')
+      # Post.create(user_id: @user, title: 'hello', text: 'Make a new comment on this post')
+      visit '/users/83'
+    end
+
+    it 'show username' do
+      expect(page).to have_content('Tom')
+    end
+
+    it 'show page title' do
+      expect(page).to have_content('User information')
+    end
+
+    it 'show the user bio' do
+      expect(page).to have_content('Teacher from Mexico')
+    end
+
+    it 'show the number of post' do
+      expect(page.body).to include('Number of post: 1')
+    end
+
+    it 'test the see all post button' do
+      expect(page).to have_content('See all post')
+    end
+
+    it 'test the create a new post' do
+      expect(page).to have_content('Create a new post')
+    end
```

Comment:
- [ ] This shouldn't not test only to have content but instead should be a test that redirects to the post's show page when you click on a user's post.